### PR TITLE
Improve changelog dialog

### LINKED ISSUE #3812: Add bug reporting instructions to patchnotes
**Describe the feature**
My suggestion (or feature request) is to add bug reporting instructions to the the top of the patchnotes that popup from the lobby whenever a new game patch is released. I often see comments & discussions of issues/bugs in several different places such as aeolus or one of the channels on Discord, but there should be a set process for players to report any issues/bugs that they encounter for the development team to review & act upon. I think by adding detailed bug reporting instructions that there will be more & better communication between the players & the development team which will make patch updates & any subsequent hotfixes much easier to handle.

**Expected behavior**
What I envision is a clear set of instructions being provided to the players with a link to the appropriate forum thread &/or specific Discord channel that they should be using to report issues/bugs as well as what info is needed such as replay#, timestamp, details of what was happening when the issue/bug appeared, etc.. These instructions should be big & bold in center alignment underneath "What's new to FAF?" & above the beginning of the patchnotes.

**Screenshots**
Patchnotes

**Additional context**
It would probably be a good idea to implement this or something similar for the FAF Develop game type so that feedback during testing will improve too.

### PR DESCRIPTION
**Pull request by Garanas**
Creates a new interface for the in-game changelog. It emphasis the number of patches by using a list instead of a combo box. The buttons link to:
 - Github: http://github.com/FAForever/fa/blob/develop/changelog.md
  - Bug report: http://github.com/FAForever/fa/wiki/report-a-bug
  - Balance notes: http://patchnotes.faforever.com/3725.html, where 3725 is replaced with the version number
  
Closes #3812 .

### CODE REVIEW

File: lua/ui/lobby/changelog.lua [hunk 1]
```diff
@@ -1,118 +1,239 @@
...
+        LayoutHelpers.FillParent(self.Background, GetFrame(0))
+
+        self.DialogBackground = Bitmap(self.CommonUI)
+        self.DialogBackground:SetSolidColor("ff111111")
+        LayoutHelpers.FillParent(self.DialogBackground, self.CommonUI)
+
+        -- Header
+
+        self.Header = Group(self.CommonUI)
+        self.Header.Left:Set(function() return self.CommonUI.Left() end)
```
Comment (4z0t) [inline lua/ui/lobby/changelog.lua:55]: self.Header.Left:Set(self.CommonUI.Left)

File: lua/ui/lobby/changelog.lua [hunk 2]
```diff
@@ -1,118 +1,239 @@
...
+-- to divide the dialog
+local debugInterface = false 
+
+Changelog = Class(Group) {
+
+    __init = function(self, parent)
+        Group.__init(self, parent)
+
+        LayoutHelpers.SetDimensions(self, 1000, 700)
+        LayoutHelpers.AtCenterIn(self, GetFrame(0))
```
Comment (4z0t) [inline lua/ui/lobby/changelog.lua:29]: Put at center of parent

**Comment by 4z0t** [comment]
Actually wanna see Layouter in action here 😄

**Comment by Garanas** [comment]
Too late - already wrote it and soonTM shipping it 📦

File: lua/ui/lobby/changelog.lua [hunk 3]
```diff
@@ -68,9 +68,9 @@ Changelog = Class(Group) {
         -- header
 
         self.Header = Group(self.CommonUI)
-        self.Header.Left:Set(function() return self.CommonUI.Left() end)
-        self.Header.Top:Set(function() return self.CommonUI.Top() end)
-        self.Header.Right:Set(function() return self.CommonUI.Right() end)
+        self.Header.Left:Set(self.CommonUI.Left())
+        self.Header.Top:Set(self.CommonUI.Top())
```
Comment (4z0t) [inline lua/ui/lobby/changelog.lua:75]: pass a function not value!

Comment (4z0t) [reply lua/ui/lobby/changelog.lua:75]: https://github.com/FAForever/fa/pull/3833#discussion_r862380136

Comment (Garanas) [reply lua/ui/lobby/changelog.lua]: Like this?

Comment (4z0t) [reply lua/ui/lobby/changelog.lua]: Yeah, unless it may(must) crush

Comment (Garanas) [reply lua/ui/lobby/changelog.lua]: I'm at a loss - what do you mean?

Comment (4z0t) [reply lua/ui/lobby/changelog.lua]: I mean, it would break 🙂

Comment (Garanas) [reply lua/ui/lobby/changelog.lua]: In what scenario would it break? I am referring to the changes in [496fd64](https://github.com/FAForever/fa/pull/3833/commits/496fd640c8e956244c647bc3143acf2aad2da172).

**Comment by 4z0t** [comment]
Looking good, but after I add Layouter, I will clean it up 😁

**Comment by Garanas** [comment]
Sounds good 😄 .

I really like the new structure of writing UI code that you introduced - it is a lot easier to keep everything clean.

**Comment by Garanas** [comment]
I'll be toying with this pull request a bit further to make sure it doesn't crash somehow and then I'll merge it.

File: lua/ui/lobby/changelog.lua (hunk 3 above)
Comment (4z0t) [reply lua/ui/lobby/changelog.lua]: Now it is ok, but before you were passing result of position calculation, that couldn't be done due to circular dependency

**Pull request merged**